python/src/RatioOfUniforms_doc.i.in

```diff
@@ -0,0 +1,487 @@
+    Distribution to sample.
+    
+    With that constructor, the range is the distribution range and *isScaled* = True.
+
+logUnscaledPDF : :class:`~openturns.Function`
+   Function :math:`\log F` such that :math:`f = F\, 1_{I}`  is proportional to the PDF :math:`p` we want to sample.
+
+rangeF : :class:`~openturns.Interval`
+    Interval :math:`I \subset \Rset^\inputDim` outside which :math:`p=0`.
+
+isScaled : bool, optional
+    Flag telling if the input *logUnscaledPDF* is such that :math:`f = F\, 1_{I}` is an actual PDF of a distribution.
+    
+    Default value is False.
+
+Notes
+-----
+This algorithm samples a multivariate distribution whose probability density probability is known up to a multiplicative factor, see [stadlober1990]_ for the unidimensional case and [gobet2016]_ for the multivariate case.
+
+Let :math:`p` be a probability density function of dimension :math:`\inputDim` and
+:math:`f: \Rset^\inputDim \rightarrow \Rset^+` a function such that:
+
+.. math::
+
+    p(\vect{x}) = cf(\vect{x})\quad  \forall \vect{x} \in \Rset^\inputDim
+    
+with :math:`c \in \Rset^+_*`. We can write :math:`f` as:
+
+.. math::
+
+    f(\vect{x}) = F(\vect{x})\, 1_{I}\quad \forall \vect{x}  \in I
+
+with :math:`I \subset \Rset^\inputDim`. It implies that the numerical range of :math:`p` is inluced in :math:`I`. We note:
+
+.. math::
+
+   I = \prod_{i=1}^\inputDim \left[a_i, b_i \right]
+
+If the function :math:`f` is an actual PDF and :math:`I` its numerical range, then :math:`c=1`.
+In the general case, :math:`c` is unknown and assumed to be difficult to compute.
+
+Let :math:`r \in \Rset^+_*`. By default, :math:`r=1`. Let :math:`A_{f,r}` be the domain defined by:
+
+.. math::
+
+    A_{f,r} & = \left\{ (u, v_1, \dots, v_\inputDim) \in \Rset^{\inputDim+1}\, |\, 0 \leq u \leq f\left(\dfrac{v_1}{u^r},
+    \dots, \dfrac{v_\inputDim}{u^r}\right)^{\frac{1}{1+r\inputDim}}\right \}
+
+The Lebesgue measure of :math:`A_{f,r}`  is equal to :math:`\dfrac{1}{c(1+rd)}`.
+
+Let :math:`(U, V_1, \dots, V_\inputDim)` be a random variable uniformly distributed on  :math:`A_{f,r}`.
+Then, :math:`\left( \dfrac{V_1}{U^r}, \dots, \dfrac{V_\inputDim}{U^r}\right)` is a random vector
+distributed according to :math:`p`.
+
+Under the condition that the functions :math:`\vect{x} \rightarrow f(\vect{x})^{\frac{1}{1+r\inputDim}}` and
+:math:`\vect{x} \rightarrow x_if(\vect{x})^{\frac{1}{1+r\inputDim}}` for all :math:`i` are bounded, then the domain
+:math:`A_{f,r}` is bounded and can be included in a rectangular bounding box :math:`\tilde{A}_{f,r}`
+defined by:
+
+.. math::
+    :label: tildeArf_def
+
+    \tilde{A}_{f,r} & = \left[0, \sup_{\vect{x}} f(\vect{x})^{\frac{1}{1+r\inputDim}} \right] \times \prod_i
+    \left[ \inf_{\vect{x}} x_i f(\vect{x})^{\frac{r}{1+r\inputDim}} ,  \sup_{\vect{x}} x_i f(\vect{x})^{\frac{r}
+    {1+r\inputDim}}\right]\\
+                    & = [0, u_{sup}] \times \prod_{i=1}^d [v_{inf, i}, v_{sup, i}]
```

Comment:
I guess that u_{sup} = \sup_{\vect{x}} f(\vect{x})^{\frac{1}{1+r\inputDim}

maybe we need to define the notation u_{sup}, v_{inf, i}, v_{sup, i} or at least say that these terms directly correspond to the above equation terms ?

Reply:
yes. I precised the notations.